client/src/components/v2/OverlaySelection.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import { action } from '@storybook/addon-actions';
+import React from 'react';
+import { addArgs, createStory } from '../../utils/stories';
+import OverlaySelection, { OverlaySelectionProps } from './OverlaySelection';
+
+export default {
+    component: OverlaySelection,
+    title: 'OverlaySelection'
+};
+
+const Template = addArgs<OverlaySelectionProps>((props) => <OverlaySelection {...props} />);
+
+const setActiveOverlay = action('set');
+
+/* ----------------------------------- Stories ----------------------------------- */
+
+export const Waiting = createStory(Template, {
+    device: 'primary',
+    setActiveOverlay,
```

Comment:
I reckon it'll be good to make some sort of base prop object, containing the props common to all examples. i.e.
```typescript
const base = {
    device: 'primary',
    setActiveOverlay,
}
export const Waiting = createStory(Template, {
    ...base,
    overlays: null
});
```
I think it makes it clearer that the only difference in the stories are in `overlays`. Also might be easier to maintain.

Reply:
Done
1847dd4

Reply:
Done
1847dd4